deeprank/generate/DataGenerator.py

```diff
@@ -411,11 +413,11 @@ def create_database(
                     self.f5.copy(mol_name + '/features/', molgrp)
 
                     # rotate the feature
-                    self._rotate_feature(molgrp, axis, angle, mol_center)
+                    DataGenerator._rotate_feature(molgrp, axis, angle, mol_center)
```

Comment:
why not self here ?

Reply:
I misunderstood static method before. self is better here.